controllers/account/ec2.go

```diff
@@ -434,6 +436,7 @@ func cleanFedrampInitializationResources(reqLogger logr.Logger, client awsclient
 	if err != nil {
 		return cleaned, err
 	}
+	reqLogger.Info("Retrieved resources", "vpc", result, "account", accountName, "region", region)
```

Comment:
I assume this is just going to log the entire string representation of the struct. Maybe we want to do something like:

- Say how many VPCs we found
- Include a `strings.Join()` representation of each VPC ID in a single comma separated string or something